src/styles/settings/_colors.scss

```diff
@@ -1,6 +1,7 @@
 $colors: (
   'white': #fff,
   'off-white': #D8D8D8,
+  'gray': #d8d8d8,
```

Comment:
'gray' is the same color as off-white here. probs don't need both?

Reply:
Oh, I change in last minute and forgot to check :( thanks for pointing out !